### ISSUE TITLE
typeString() func it's not work with Electron in macOS

### ISSUE THREAD

**Issue by wallezhang** [octalmage/robotjs#619]
## Expected Behavior
I register a global shortcut in electron, and in callback call `robot.typeString()` function. But it seems not work, nothing to be typed. If i call `robot.keyTap()` function first, and then call `robot.typeString()` function, that will type a truncate string at first time, and i press the global shortcut again, then it types normal string what i want.

## Steps to Reproduce (for bugs)
Here is an example code
```javascript
    globalShortcut.register('CommandOrControl+Alt+Shift+n', () => {
        console.log('CommandOrControl+Alt+Shift+n is pressed');
        robot.typeString('Hello World');
    });
```
Above code is not work when i press the shortcut

And if i call `robot.keyTap()` first
```javascript
    globalShortcut.register('CommandOrControl+Alt+Shift+n', () => {
        console.log('CommandOrControl+Alt+Shift+n is pressed');
        robot.keyTap('enter');
        robot.typeString('Hello World');
    });
```

When i press shortcut first, it will type an enter without any string, and then i press shortcut again, it will work.

## Your Environment

* RobotJS version: 0.6.0
* Node.js version: 15.0.1
* Electron version: 10.1.2
* Operating System: macOS Catalina 10.15.6

**Comment by A1exXxe1A** [comment]
try another key combination, like "Alt+Shift+n"

**Comment by wallezhang** [comment]
> try another key combination, like "Alt+Shift+n"

Yes! It worked!! I found key combination with `Command` or `Control` will not work. Is there a conflict?

**Comment by wallezhang** [comment]
BTW, `robot.keyTap()` function is work as expect whatever key combination i use.

**Comment by octalmage** [comment]
Closing stale issues as part of a cleanup. If this is still an issue, please reopen and I'll take a look. Thanks!
